README.md

```diff
@@ -109,8 +109,8 @@ Set `DEPLOY_UPLOAD_IMAGES=true` if you modified your guest and need to upload a
 RISC0_DEV_MODE=true DEPLOY_RELAY_ADDRESS="$APP_ADDRESS" DEPLOY_UPLOAD_IMAGES=true forge script script/Deploy.s.sol --rpc-url http://localhost:8545 --broadcast
 ```
 
-This will deploy only your application address and upload any updated images.
-The existing relay contract and, by setting `DEPLOY_RELAY_ADDRESS`, the running relay will continue to be used.
+This will deploy only your application address and upload any updated images. 
+By setting `DEPLOY_RELAY_ADDRESS`, the existing running relay contract will continue to be used.
```

Comment:
It's not just the that the same contract will be used, there is a relay running and it will continue to be used if specified with this environment variable.
```suggestion
By setting `DEPLOY_RELAY_ADDRESS`, the existing running relay contract and the already running relay will continue to be used.
```